client/my-sites/plans/current-plan/index.jsx

```diff
@@ -33,14 +33,14 @@ class CurrentPlan extends Component {
 	static propTypes = {
 		selectedSiteId: PropTypes.number,
 		selectedSite: PropTypes.object,
-		sitePlans: PropTypes.object,
+		isRequestingSitePlans: PropTypes.bool,
 		context: PropTypes.object
 	};
 
 	isLoading() {
-		const { selectedSite, sitePlans } = this.props;
+		const { selectedSite, isRequestingSitePlans: isRequestingPlans } = this.props;
 
-		return ! selectedSite || ! sitePlans.hasLoadedFromServer;
+		return ! selectedSite || isRequestingPlans;
```

Comment:
Oh whoops I misread the requesting flag, and I forgot how weird the plans tree was. It's not persisted due to the moment instances, and the request status is tangled in the data. In the future we can eventually just check for the presence of the plan values, but it'd require decoupling the tree into plan 'items' and a 'requesting' sub tree.
```
return ! selectedSite || ! sitePlans;
```